website/package.json

```diff
@@ -11,11 +11,11 @@
     "graphql-markdown": "^5.2.0"
   },
   "dependencies": {
-    "docusaurus2-graphql-doc-generator": "file:docusaurus2-graphql-doc-generator",
-    "@docusaurus/core": "^2.0.0-alpha.70",
-    "@docusaurus/preset-classic": "^2.0.0-alpha.70",
-    "@docusaurus/types": "^2.0.0-alpha.70",
+    "@docusaurus/core": "==2.0.0-alpha.70",
+    "@docusaurus/preset-classic": "==2.0.0-alpha.70",
+    "@docusaurus/types": "==2.0.0-alpha.70",
```

Comment:
Why? There's a lock file.

Reply:
with `^` pre-release versions were updating in lock. And those had breaking changes :/ 
Anyway, cant find issue why those builds are failing on CI, tried to clean my node_modules and create lock again, and no luck with that. Node version freeze was blind try to find whats different about my local env and CI

Reply:
The lock file does not update itself unless you explicitly tell npm to upgrade a dependency (in which case you are responsible for verifying its compatibility).